src/days-2-4-adv/adv.py

```diff
@@ -33,19 +35,82 @@
 room['narrow'].n_to = room['treasure']
 room['treasure'].s_to = room['narrow']
 
+# item = {
+#     'sword':  Item("sword",
+#                      "slay the creeps using the sword."),
 #
-# Main
+#     'ax':    Item("ax",
+#                         """crush them with the ax"""),
 #
-
-# Make a new player object that is currently in the 'outside' room.
-
-# Write a loop that:
-#
-# * Prints the current room name
-# * Prints the current description (the textwrap module might be useful here).
-# * Waits for user input and decides what to do.
+#     'torch': Item("torch",
+#                         """Find your way!"""),
 #
-# If the user enters a cardinal direction, attempt to move to the room there.
-# Print an error message if the movement isn't allowed.
+#     'amulet':   Item("amulet",
+#                          """Stop spells"""),
 #
-# If the user enters "q", quit the game.
+# }
+
+coins = Treasure("coins", "gold coins for the taking")
+rubies = Treasure("rubies", "sparkleys")
+
+sword = Item("sword", "slay the creeps using the sword.")
+ax = Item("ax", "crush them with the ax")
+mace = Item("mace", "fly anything with the mace")
+
+
+
+playerStartingItems = [sword]
+room['outside'].addItem(ax)
+room['foyer'].addItem(mace)
+room['treasure'].addItem(coins)
+room['treasure'].addItem(rubies)
+
```

Comment:
Did this work when you tried it? I am curious because I think in room.py, you init your rooms with self, name, description, and items as parameters. I think you would need to add the items to the end of each room declaration in lines 7-24. Otherwise, I'm pretty sure you'll get an error saying that you're only passing 2 parameters when you need 3.